components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -192,7 +381,13 @@ DrawerNavGroup.propTypes = {
     ).isRequired,
     onSelect: PropTypes.func,
```

Comment:
I'm this far into the code review and I'm still confused about what onSelect means.

Reply:
Resolve: logged an issue https://github.com/pxblue/react-component-library/issues/28